lib/src/pages/layouts/yaru_master_detail_page.dart

```diff
@@ -32,6 +34,9 @@ class YaruMasterDetailPage extends StatefulWidget {
     required this.titleBuilder,
     required this.pageBuilder,
     required this.leftPaneWidth,
+    this.leftPaneResizing = true,
```

Comment:
nit-pick: this sounds like the pane is currently resizing :)

Reply:
Hmm, what about `allowLeftPaneResize`?

Reply:
:+1: I'm not a native English speaker either but it sounds better in my head :slightly_smiling_face: